Imager.js

```diff
@@ -142,30 +143,36 @@
           });
         }
 
-        if (elements) {
-            this.divs = applyEach(elements, returnFn);
-            this.selector = null;
-        }
-        else {
-            this.divs = applyEach(doc.querySelectorAll(this.selector), returnFn);
-        }
-
+        this.divs = [];
+        this.add(elements || this.selector);
         this.ready(opts.onReady);
-        this.changeDivsToEmptyImages(this.divs);
 
-        setTimeout(function(){
+        setTimeout(function () {
             self.init();
         }, 0);
-    }
+    };
+
+    Imager.prototype.add = function (elementsOrSelector) {
+
+        elementsOrSelector = elementsOrSelector || this.selector;
+        var elements = typeof elementsOrSelector === 'string' ?
+            document.querySelectorAll(elementsOrSelector) : // Selector
+            elementsOrSelector; // Elements (NodeList or array of Nodes)
+
+        if (elements.length) {
+            this.divs = this.divs.concat(applyEach(elements, returnFn));
```

Comment:
I would apply `changeDivsToEmptyImages` only to the new divs, just to sake of saving some iterations.